src/latest/lib/dsl.sc

```diff
@@ -246,6 +245,25 @@ class DSL(implicit val mat: Materializer, timeout: Timeout) {
     purge(Seq(value))
   }
 
+  def migrate()(implicit module: StorageToolModule): Seq[String] = {
```

Comment:
Are you able to actually run migrations like this? We assert that the version matches the target tool version before we even allow the repl to be used.

Reply:
Yes, it worked for me, though I am planning to give another round of testing today or tomorrow. The reason is the assertion
```
$ rg assertStoreCompat
bindings.sc
104:  def assertStoreCompat: Unit = {
```
doesn't seem to be used.

Reply:
how did you launch the tool?

Reply:
It is used by the launcher script, which is consequently used as the docker entry point:

https://github.com/mesosphere/marathon-storage-tool/blob/master/src/1.4.x/bin/storage-tool.sh